action.yml

```diff
@@ -127,7 +127,7 @@ runs:
       if: inputs.reproducible == 'assert' || inputs.reproducible == 'report'
       shell: bash
       working-directory: ${{ inputs.work_dir }}
-      run: $RUN_CLI build -verification
+      run: $RUN_CLI build -clean -json -verification
```

Comment:
Same two flags added as above. The `-verification` flag makes this a verification build.